src/component/context/ContextForm/ContextForm.tsx

```diff
@@ -45,54 +47,64 @@ export const ContextForm: React.FC<IContextForm> = ({
 }) => {
     const styles = useStyles();
     const [value, setValue] = useState('');
-    const [focused, setFocused] = useState(false);
+    const [valueDesc, setValueDesc] = useState('');
+    const [valueFocused, setValueFocused] = useState(false);
 
-    const submit = (event: React.SyntheticEvent) => {
+    const isMissingValue = valueDesc.trim() && !value.trim();
+
+    const isDuplicateValue = legalValues.some(legalValue => {
+        return legalValue.value.trim() === value.trim();
```

Comment:
This causes a crash when opening existing context fields without legal values:

<img width="1202" alt="Skjermbilde 2022-04-19 kl  11 12 09" src="https://user-images.githubusercontent.com/16081982/163971936-895dbdf9-da86-4279-b9fa-7e2d976b287c.png">